src/test/java/racingcar/CarListTest.java

```diff
@@ -0,0 +1,28 @@
+package racingcar;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarListTest {
+
+    @DisplayName("자동차 경주 리스트를 초기화한다")
+    @Test
+    void initialize() {
+        CarList cars = CarList.initialize(3);
+
+        assertThat(cars.currentCars().size()).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(cars.currentCars()).hasSize(3);
```
Array에 대해서 검증할 수 있는 메서드들도 assertJ에서 제공하는게 있어서 이를 활용해봐도 좋을거 같아요 :)

참고: https://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#filters

Reply:
좋은 메소드군요 알아갑니다 감사합니다 !!